app/views/ncr/proposals/new.html.erb

```diff
@@ -32,6 +32,15 @@
       <% end %>
     </div>
     <div class="form-group">
+      <%= f.label :office %>
+      <%= f.select(:office, [], {}, {class: 'form-control', multiple: false}) do %>
+        <%= content_tag(:option, '', value: '') %>
+        <% Ncr::ProposalForm::OFFICES.each do |office| %>
+          <%= content_tag(:option, office, value: office) %>
+        <% end %>
+      <% end %>
```

Comment:
I *think* this could all be simplified to

```erb
<%= f.collection_select :office, Ncr::ProposalForm::OFFICES, :to_s, :to_s, {include_blank: true}, {class: 'form-control'} %>
```